src/workloads/scale/LargeScaleModel.yml

```diff
@@ -0,0 +1,469 @@
+
+# This section contains shared definitions that are used in the workload.
+GlobalDefaults:
+  Random10KInt: &rand_10k_int {^RandomInt: {min: 0, max: 10000}}
+  Random10KInt: &rand_1k_int {^RandomInt: {min: 0, max: 1000}}
+  Random4ByteInt: &rand_4b_int {^RandomInt: {min: 0, max: 2147483647}}
+  Random30String: &rand_30b_string {^RandomString: {length: 30}}
+  Random130String: &rand_100b_string {^RandomString: {length: 100}}
+
+  # CHANGE: set to 300 minutes or longer
+  Duration: &Duration 70 minutes   # set low, so we can see the effect of scans
+  TrackProportion: &TrackProportion 0
+
+  LongLivedDB: &LongLivedDB longlived  # name of the long lived database
+  LongLivedDocumentCount: &LongLivedDocumentCount 1000
+  LongLivedCollectionCount: &LongLivedCollectionCount 1000
+
+  RollingDB: &RollingDB rolling
+
+  # Note: if HotCollectionDB and HotDocumentDB reference the same
+  # database name, only one of their loaders should be enabled.
+  HotCollectionDB: &HotCollectionDB hotcoll
+  HotDocumentDB: &HotDocumentDB hotdoc
+
+  ScannerColdDB: &ScannerColdDB scancold
+  ScannerHotDB: &ScannerHotDB rolling
+
+  # Set to how many gigabytes we want the "cold" database to be.
+  # This database is initially created, but not used for anything,
+  # except for the snapshot scanner that scans all databases.
+  # We expect this to be larger than the amount of physical memory,
+  # so that we'll displace lots of hot data in the cache.
+  # CHANGE: the spec says 10TB of live data (total in all databases).
+  ScannerColdDBGigabytes: &ScannerColdDBGigabytes 150
+
+  # This list is used to scan all databases. By putting first the hot databases,
+  # which have high concentrations of updates, we'll be referencing more
+  # and "older" updates. That should stress the WiredTiger version history.
+  AllDB: &AllDB hotdoc,hotcoll,rolling,longlived,scancold
+
+  # According to the design, the number of writes should add up to 10K second.
+  LongLivedWrites: &LongLivedWrites 600 per 1 second          #  x5 threads
+  RollingWrites: &RollingWrites 1000 per 1 second             # x40 threads
+  HotDocumentWrites: &HotDocumentWrites 1000 per 1 second     # x40 threads
+  HotCollectionWrites: &HotCollectionWrites 1000 per 1 second # x40 threads
+
+  # According to the design, the number of reads should add up to 1K second.
+  LongLivedReads: &LongLivedReads 20 per 1 second            # x10 threads
+  LongLivedIndexReads: &LongLivedIndexReads 20 per 1 second  # x10 threads
+  RollingReads1: &RollingReads1 50 per 1 second              # x10 threads
+  RollingReads2: &RollingReads2 100 per 1 second             # x1 thread
+
+  # We want about 200 bytes, 9 fields (they will all be indexed).
+  RollingDocument: &RollingDocument {
+    a: *rand_30b_string,
+    b: *rand_30b_string,
+    c: *rand_30b_string,
+    d: *rand_30b_string,
+    e: *rand_30b_string,
+    f: *rand_30b_string,
+    g: *rand_4b_int,
+    h: *rand_4b_int,
+    i: *rand_4b_int }
```

Comment:
you don't need the curly brackets or the commas:

```yaml
Rolling: &Rolling
  a: *rand
  b: *rand
```

Reply:
Great, fixed.